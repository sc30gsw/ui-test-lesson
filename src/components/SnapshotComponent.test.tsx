import { render } from '@testing-library/react'

import SnapshotComponent from './SnapshotComponent'

describe('SnapshotComponent', () => {
  it('Snapshotテスト', () => {
    const { container } = render(<SnapshotComponent text="Next.js" />)
    expect(container).toMatchSnapshot()
  })
})
