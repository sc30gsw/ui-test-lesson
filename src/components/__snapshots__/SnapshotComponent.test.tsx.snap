// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SnapshotComponent Snapshotテスト 1`] = `
<div>
  <div>
    <p>
      Next.js
    </p>
  </div>
</div>
`;
